pkg/config/remote/service/util.go

```diff
@@ -111,12 +111,12 @@ func openCacheDB(path string, agentVersion string, apiKey string, url string) (*
 		bucket := tx.Bucket([]byte(metaBucket))
 		if bucket == nil {
 			log.Infof("Missing meta bucket")
-			return err
+			return log.Errorf("Could not get RC metadata: missing bucket")
```

Comment:
I think this should be `fmt.Error()` as it's logged as `Info` level right after (line 130) and we return an error at line 131 if the recreation failed. Also the data returned will be missing the info about the actual name of the bucket or file getting read.